lib/internal/test_runner/harness.js

```diff
@@ -252,7 +252,11 @@ function lazyBootstrapRoot() {
       __proto__: null,
       entryFile: process.argv?.[1],
     };
-    const globalOptions = parseCommandLine();
+    const globalOptions = {
```

Comment:
I wonder if it would be more efficient to either:

1. Add `cwd` to the object returned in `parseCommandLine()`. Although then that function is not strictly only working with the command line.
2. Do `globalOptions.cwd = process.cwd()` here instead of creating an object in `parseCommandLine()`, spreading it here, and creating a new object containing all the same values as the original object.

I'm personally leaning toward option 2.

Reply:
hey @cjihrig, I agree regarding option 2.
IMHO option 1 would make sense if we were adding a new option.

I'm going to take a look ASAP